src/current/cockroachcloud/resource-usage.md

```diff
@@ -27,7 +27,7 @@ CockroachDB {{ site.data.products.standard }} clusters consume three kinds of re
 
 A CockroachDB {{ site.data.products.standard }} cluster is divided into a SQL layer and a storage layer that run in separate processes. The SQL layer receives and runs your SQL queries and background jobs. When the SQL layer needs to read or write data rows, it calls the storage layer, which manages a replicated, transactional row store that is distributed across many machines.
 
-**SQL CPU** is the CPU consumed by SQL processes (not storage processes) and is converted to [Request Units]({% link cockroachcloud/plan-your-cluster.md %}#request-units) using this equivalency: 1 RU = 3 milliseconds SQL CPU.
+**SQL CPU** is the CPU consumed by SQL processes (not storage processes) and is converted to [Request Units]({% link cockroachcloud/plan-your-cluster-basic.md %}#request-units) using this equivalency: 1 RU = 3 milliseconds SQL CPU.
```

Comment:
This links to the Basic page just after talking about Standard.

Reply:
Leaving this one alone because the page needs so many updates and I don't want to break the link. I think this page needs to become a page about Basic, and we need new content for the other tiers.